src/client/containers/404Page/404Page.css

```diff
@@ -0,0 +1,83 @@
+.body-background-image {
+  background-image: url(../../assets/images/Vector.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.center-container {
+  width: 45%;
+  height: 65%;
+  margin: 9% auto;
+  background: rgba(179, 64, 74, 0.73);
+  position: relative;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.center-container img {
+  margin: 1rem 1rem;
+  width: 7rem;
+  height: 7rem;
+}
+
+.main-text {
+  position: absolute;
+  top: 25%;
+  left: 33%;
+}
+
+.main-text h1 {
+  font-family: 'Chakra Petch', sans-serif;
+  letter-spacing: 0.05rem;
+  font-weight: bold;
+  font-style: normal;
+  font-size: 7rem;
+  margin: 0;
+  color: #000;
+  text-shadow: 0 8px 4px rgba(0, 0, 0, 0.25);
+}
+
+.main-text p {
+  text-transform: uppercase;
+  font-weight: bold;
+  margin: 0;
+  color: #000;
+  text-align: center;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.circle-container {
+  background-color: #d9e4e3;
+  border-radius: 50%;
+  height: 13rem;
+  width: 13rem;
+  position: absolute;
+  left: 80%;
+  top: 20%;
+}
+
+.circle-container img {
+  margin-top: 1.5rem;
+  margin-left: 3rem;
+}
+
+.circle-container p {
+  text-transform: uppercase;
+  text-align: center;
+  width: 80%;
+  margin-top: -1.3rem;
+  margin-left: 1.3rem;
+  color: #323334;
+}
+
+.center-container .buttons {
+  padding: 12px 50px;
+  position: absolute;
+  bottom: -5%;
+  left: 35%;
```

Comment:
Be careful with absolute position that should be your last way out to style it because it will in a lot of case not be very responsive for different screensizes